kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/library/maven/search/SearchMavenDialogController.java

```diff
@@ -31,15 +31,6 @@
  */
 package com.oracle.javafx.scenebuilder.kit.editor.panel.library.maven.search;
```

Comment:
Please update license header here in line 2 from `* Copyright (c) 2016, 2017 Gluon and/or its affiliates.` to `* Copyright (c) 2016, 2022 Gluon and/or its affiliates.`.